docs/source/contents/4_draft_mapping.rst

```diff
@@ -118,21 +118,143 @@ We have decided to not migrate these values as is justified above. Here's an `ex
+
+Xpath
+^^^^^
+mods:physicalDescription/mods:note
+
+Decision
+^^^^^^^^
+All values will be moved to a generic note field.
+
+`Example record - derris:879 <https://digital.lib.utk.edu/collections/islandora/object/derris%3A879/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <physicalDescription>
+        <form authority="aat" valueURI="http://vocab.getty.edu/aat/300127478">transparencies</form>
+        <digitalOrigin>digitized other analog</digitalOrigin>
+        <note>Film type: GEMounts</note>
+        <note>Camera setting: 10@50 at 4ft</note>
+    </physicalDescription>
+
+.. code-block:: turtle
+
+    @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+    <https://example.org/objects/1>
+        skos:note "Film type: GEMounts", "Camera setting: 10@50 at 4ft" .
 
 extent
 ------
 
 Use Case
 ^^^^^^^^
+The element includes values that indicate time and physical dimensions. Time is consistently shared in hours, minutes
+and seconds. Physical dimensions are most consistently represented in inches and feet, but cm are also used for smaller
+items that might benefit from a more granular measurement. While this kind of information has historically been included
+in MARC records to ensure that books are not larger than the shelf height, extent values can also provide important
+contextual information that is relevant to better understanding resources in a digital environment. Particularly in the
+case of photography, the dimensions can be used to help determine the type of film.
+
+Justification
+^^^^^^^^^^^^^
+The working group's shared philosophies were influential in decided on the best property to use for <extent> values. The
+Islandora Metadata Interest Group's default mapping suggests using dcterms:extent and using a blank node with a literal as
+a RDF value. This group is against using blank nodes when at all possible because they make it more difficult for the
+user to consume content. The Samvera mapping uses rdau:extent, which is less than ideal because rdau does not support
+content negotiation. This means that the URI provided for the desired property does not allow a user to directly request
+RDF. No other more suitable properties could be found for <extent> values. Given this predicament, the working group
+decided to use rdau:extent because it is dereferenceable, which a blank node is not. Still, the inability to retrieve
+RDF directly will limit users wishing to interact with our data in this way.
+
+Xpath
+^^^^^
+
+mods:physicalDescription/mods:extent
+
+Decision
+^^^^^^^^
+`Example record - knoxgardens:125 <https://digital.lib.utk.edu/collections/islandora/object/knoxgardens%3A125/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <extent>3 1/4 x 5 inches</extent>
+
+.. code-block:: turtle
+
+    @prefix rdau: <http://rdaregistry.info/Elements/u/#> .
+
+    <https://example.org/objects/1>
+        rdau:extent.en "3 1/4 x 5 inches" .
```

Comment:
See the message in my review.  Also, the turtle lexer does not like namespaced URIs like this. If we don't change properties, I think we should just use the full URI.